dependencies/versions.go

```diff
@@ -21,6 +21,14 @@ import (
 	"github.com/pkg/errors"
 )
 
+// VersionUpdate represents the schema of the output format
+// The output format is dictated by ValidateOptions.outputFormat
+type VersionUpdate struct {
+	Name       string `yaml:"name" json:"name"`
+	Version    string `yaml:"version" json:"version"`
+	NewVersion string `yaml:"new_version" json:"new_version"`
```

Comment:
What are your thoughts on exposing information about both next minor/next patch?

There are some comments about this here:
https://github.com/kubernetes-sigs/zeitgeist/issues/21#issue-784877780

I´m not saying this should be part of the output now but I _wonder if the schema should support it_

Data points:
* go get -u supports both minor and patch releases
* me :) sometimes I want the next/latest patch release, other times I want to bump the minor version.

Reply:
I think you can do those things with a combination of ["sensitivity"](https://github.com/kubernetes-sigs/zeitgeist/blob/master/dependencies/dependencies.go#L56) and ["constraints"](https://github.com/kubernetes-sigs/zeitgeist/blob/master/upstreams/github.go#L36).

You can use `sensitivity` to tell zeitgeist to "detect" new version only if minor (or major) changes, or only if the major changes, etc.  

Similarly you can use `constraints` to tell zeitgeist to "detect" a new version only if the new version meets the constraints.
Example: 
If your current version is 1.19.1 and you only want to detect new patch updates you can setup a constraint to "< 1.20.0" and zeitgiest will only "detect" a patch update.

Using the above 2 you can come up with a combination of component configurations that can be used to detect only the changes you are looking for. The output will then only include the "new version" that matches the criteria you are looking for.

Reply:
The `version` and `new_version` fields right now are independent of the scheme. My concern is introducing fields specific to the semver scheme will be a violation of that.

PS: zeitgeist also supports the following schemes besides semver.
* random
* alpha